src/WpfMath/TexPredefinedFormulaParser.cs

```diff
@@ -173,8 +173,11 @@ public override void Parse(SourceSpan source, XElement element)
                 {
                     formula = new TexFormula();
                 }
-
-                this.TempFormulas.Add(name, formula);
+                if (!this.TempFormulas.ContainsKey(name))
+                {
+                    this.TempFormulas.Add(name, formula);
+                }
```

Comment:
This code is supposed to fix #139 but I don't think this is the right approach: it deliberately ignores the formulas introduced by different commands and merges them into one. I don't think that's right, even if it accidentally works for simple case with `\,` → `\thinspace`.

In other words, `<CreateFormula name="f">` in `\thinspace` should have no relation to `<CreateFormula name="f">` in `\,`. Each `<Formula>` block should get its own name scope.